src/ImageList/manifest.json

```
@@ -284,6 +303,282 @@
+        },
+        {
+          "name": "borderColor",
+          "displayName": "Border Color",
+          "type": "color",
+          "default": "#E0E0E0",
+          "enabled": {
+            "searchInputStyles": true
+          }
+        },
+        {
+          "name": "borderSize",
+          "displayName": "Border Size",
+          "type": "number",
+          "default": 1,
+          "control": { "type": "slider", "max": 18, "min": 0 },
+          "enabled": {
+            "searchInputStyles": true
+          }
+        },
+        {
+          "name": "rounding",
+          "displayName": "Rounding",
+          "type": "number",
+          "default": 4,
+          "control": { "type": "slider", "max": 18, "min": 0 },
+          "enabled": {
+            "searchInputStyles": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "listEmptyState",
+      "displayName": "List Empty State",
+      "props": [
+        {
+          "name": "emptyStateImageStatus",
+          "displayName": "Image",
+          "type": "text",
+          "default": "noImage",
+          "control": {
+            "type": "menu",
+            "options": [
+              { "label": "Above Title", "value": "above" },
+              { "label": "Below Title", "value": "below" },
+              { "label": "No Image", "value": "noImage" }
+            ]
+          }
+        },
+        {
+          "name": "imageSource",
+          "displayName": "Image Source",
+          "role": "listItem",
+          "reference": "images",
+          "type": "image",
+          "enabled": {
+            "emptyStateImageStatus": ["above", "below"]
+          }
+        },
+        {
+          "name": "textTitleDisplay",
+          "displayName": "Text",
+          "type": "text",
+          "default": "titleOnly",
+          "control": {
+            "type": "menu",
+            "options": [
+              { "label": "Title Only", "value": "titleOnly" },
+              { "label": "Title and Subtitle", "value": "titleAndSubtitle" },
+              { "label": "No Text", "value": "noText" }
+            ]
+          }
+        },
+        {
+          "name": "title",
+          "displayName": "Title",
+          "type": "text",
+          "default": "No List Items",
+          "styles": {
+            "fontFamily": "@heading",
+            "fontWeight": "600",
+            "color": "@text",
+            "fontSize": "18"
+          },
+          "enabled": {
+            "textTitleDisplay": ["titleOnly", "titleAndSubtitle"]
+          }
+        },
+        {
+          "name": "subtitle",
+          "displayName": "Subtitle",
+          "type": "text",
+          "default": "Subtitle",
+          "styles": {
+            "fontFamily": "@body",
+            "color": "#212121",
+            "fontWeight": "normal"
+          },
+          "enabled": {
+            "textTitleDisplay": "titleAndSubtitle"
+          }
+        },
+        {
+          "name": "type",
```

```suggestion
          "name": "buttonType",
```
Make this change everywhere, makes things easier